src/international-phone-number.coffee

```diff
@@ -73,7 +73,6 @@ angular.module("internationalPhoneNumber", []).directive 'internationalPhoneNumb
       if value
         validity = element.intlTelInput("isValidNumber")
         ctrl.$setValidity 'international-phone-number', validity
-        ctrl.$setValidity '', validity
       else
         value = ''
```

Comment:
Yes, no problem. Please submit a pull request, I'll merge it as one of 0.0.*. Then I'll move the solution with $validators to 0.1.0 as you suggested